src/test/java/com/microsoft/azure/spring/data/documentdb/core/DocumentDbTemplateIT.java

```diff
@@ -137,54 +131,47 @@ public void testFindByIdPartition() {
     @Test
     public void testUpdate() {
         final Person updated = new Person(TEST_PERSON.getId(), "updatedname",
-                TEST_PERSON.getLastName());
+                TEST_PERSON.getLastName(), TEST_PERSON.getHobbies());
         dbTemplate.update(Person.class.getSimpleName(), updated, updated.getId(), null);
 
         final Person result = dbTemplate.findById(Person.class.getSimpleName(),
                 updated.getId(), Person.class, null);
 
-        assertThat(result.getId()).isEqualTo(updated.getId());
-        assertThat(result.getFirstName()).isEqualTo(updated.getFirstName());
-        assertThat(result.getLastName()).isEqualTo(updated.getLastName());
+        assertTrue(result.equals(updated));
     }
 
     @Test
     public void testUpdatePartition() {
         setupPartition();
         final Person updated = new Person(TEST_PERSON.getId(), "updatedname",
-                TEST_PERSON.getLastName());
+                TEST_PERSON.getLastName(), TEST_PERSON.getHobbies());
         dbTemplate.update(Person.class.getSimpleName(), updated, updated.getId(), updated.getLastName());
 
         final Person result = dbTemplate.findById(Person.class.getSimpleName(),
                 updated.getId(), Person.class, updated.getLastName());
 
-        assertThat(result.getId()).isEqualTo(updated.getId());
-        assertThat(result.getFirstName()).isEqualTo(updated.getFirstName());
-        assertThat(result.getLastName()).isEqualTo(updated.getLastName());
+        assertTrue(result.equals(updated));
     }
 
     @Test
     public void testDeleteById() {
-        final Person person2 = new Person("newid", "newfn", "newln");
+        final Person person2 = new Person("newid", "newfn", "newln", null);
         dbTemplate.insert(person2, null);
         assertThat(dbTemplate.findAll(Person.class, null, null).size()).isEqualTo(2);
 
         dbTemplate.deleteById(Person.class.getSimpleName(), TEST_PERSON.getId(), null, null);
 
         final List<Person> result = dbTemplate.findAll(Person.class, null, null);
         assertThat(result.size()).isEqualTo(1);
-        assertThat(result.get(0).getId()).isEqualTo(person2.getId());
-        assertThat(result.get(0).getFirstName()).isEqualTo(person2.getFirstName());
-        assertThat(result.get(0).getLastName()).isEqualTo(person2.getLastName());
-
+        assertTrue(result.get(0).equals(person2));
     }
 
     @Test
     public void testDeleteByIdPartition() {
         setupPartition();
 
         // insert new document with same partition key
-        final Person person2 = new Person("newid", "newfn", TEST_PERSON.getLastName());
+        final Person person2 = new Person("newid", "newfn", TEST_PERSON.getLastName(), null);
```

Comment:
why null

Reply:
Will change to some other more meaningful value.